providers/dns/exoscale/exoscale.go

```diff
@@ -5,15 +5,18 @@ import (
 	"context"
 	"errors"
 	"fmt"
-	"net/http"
 	"time"
 
-	"github.com/exoscale/egoscale"
+	egoscale "github.com/exoscale/egoscale/v2"
 	"github.com/go-acme/lego/v4/challenge/dns01"
 	"github.com/go-acme/lego/v4/platform/config/env"
 )
 
-const defaultBaseURL = "https://api.exoscale.com/dns"
+// Defaults.
+const (
+	defaultBaseURL = "https://api.exoscale.com/v2"
+	defaultAPIZone = "ch-gva-2"
```

Comment:
can you explain this default value?

Reply:
Commented code but here is a longer explanation: API Zone is [data center location](https://www.exoscale.com/datacenters/), each DC hosts API. Important for other API endpoints, not so much for DNS but API expects it. Actual endpoint is built as `https://<api-zone>-<base-url>`, for default values it would be: `https://ch-gva-2-api.exoscale.com/v2`. That default value (Geneva, CH) is just convention we have in Exoscale (`api.exoscale.com` is alias to it).